packages/client/src/components/form/LinkButton.tsx

```diff
@@ -94,7 +94,7 @@ export const LinkButtonField = ({
       disabled={isDisabled}
       onClick={() => {
         window.location.href = evalExpressionInFieldDefinition(
-          '`' + to + '`',
+          '`' + decodeURIComponent(to) + '`',
```

Comment:
So the url comes as urlencoded from the form config?

Reply:
Yea, causing problems with dollar signs